### PR TITLE
Performax Bid Adapter: New bidder adapter

### PR DESCRIPTION
**Pull request by lukashakl**
## Type of change

- [x] New bidder adapter  

## Description of change
New bidder adapter

<!-- For new bidder adapters, please provide the following
Maintainer: development@performax.cz
- test parameters for validating bids:
```
 {
                    bidder: "performax",
                    params: {
                        slotId: "sample"   // required
                    }
                }
```

### CODE REVIEW

**Comment by ncolletti** [comment]
@lukashakl could you link your prebid website docs PR?

**Comment by lukashakl** [comment]
> @lukashakl could you link your prebid website docs PR?
Sorry, there it is https://github.com/prebid/prebid.github.io/pull/5259

**Comment by ncolletti** [comment]
@lukashakl Sorry for the delay in the review, looks like your PR doesn't have unit tests committed. Could you add those?

File: modules/performaxBidAdapter.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        {
+            code: 'performax-div',
+            mediaTypes: {
+                banner: { sizes: [[300, 300]]},
+           
+            bids: [
+                {
+                    bidder: "performax",
+                    params: {
+                        slotId: "sample"   // required
```
[the remark this answers is not in the document]
Comment (lukashakl) [reply modules/performaxBidAdapter.md]: sorry that was old version. Now there is a tagid. This example should return valid bid response.

**Comment by lukashakl** [comment]
> @lukashakl Sorry for the delay in the review, looks like your PR doesn't have unit tests committed. Could you add those?

Thanks for response. I add unit test today. It´s ok now?

**Comment by lukashakl** [comment]
@ncolletti do you have any idea why this CI failed? I haven´t found anything related to my request in logs and I´m not able re-run this tests.
image

File: modules/performaxBidAdapter.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ncolletti) [reply modules/performaxBidAdapter.md]: I'm seeing 400 responses from the example params in the PR, can you take a look?

**Comment by ncolletti** [comment]
@lukashakl there are some flaky integration tests that can fail, unrelated to your changes

File: modules/performaxBidAdapter.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lukashakl) [reply modules/performaxBidAdapter.md]: Hi,

I've fixed the brackets in the example. According to our logs, there were no requests on 05-02-2024. We only received requests from our internal testing (from 05-03-2024), and none from outside the company.

Could you test these changes, and if they do not work, could you send us the request you attempted to post to our endpoint?

[the remark this answers is not in the document]
Comment (ncolletti) [reply modules/performaxBidAdapter.md]: This is using the "sample" tagid from your bidder docs, we expect that id to return a valid bid response so we can test your adapter

[the remark this answers is not in the document]
Comment (lukashakl) [reply modules/performaxBidAdapter.md]: The chip.performax.cz... endpoint is only for our statistics counting and not affecting response itself

[the remark this answers is not in the document]
Comment (lukashakl) [reply modules/performaxBidAdapter.md]: Hi,
We have fixed the response from the chip.performax.cz endpoint, and it no longer returns an error.

**Review by ncolletti — APPROVED** [review]
LGTM

**Pull request merged**